internal/locate/region_cache.go

```diff
@@ -755,6 +755,40 @@ func (l *KeyLocation) GetBucketVersion() uint64 {
 	return l.Buckets.GetVersion()
 }
 
+// LocateBucketV2 will not return nil if the key is in the region.
+// LocateBucketV2 is similar with LocateBucket. The difference is that when the key is in [KeyLocation.StartKey, first Bucket key)
+// it will return Bucket{KeyLocation.StartKey, first Bucket key} rather than nil --- it's reasonable to assume that
+// Bucket{KeyLocation.StartKey, first Bucket key} is a bucket belonging to the region. Key in [last Bucket key, KeyLocation.EndKey)
+// is handled similarly.
+func (l *KeyLocation) LocateBucketV2(key []byte) *Bucket {
```

Comment:
What does `V2` suppose to mean? Does that mean there would be two versions of the `LocateBucket` function and they would be used in different situations? If so it's better to change the name to some meaningful name but not use `V2`.

Reply:
/cc @SpadeA-Tang